protocol-units/execution/opt-executor/src/executor/transaction_pipe.rs

```diff
@@ -45,10 +44,17 @@ impl Executor {
 			match request {
 				MempoolClientRequest::SubmitTransaction(transaction, callback) => {
 					// For now, we are going to consider a transaction in flight until it exits the mempool and is sent to the DA as is indicated by WriteBatch.
-					let in_flight = self.transactions_in_flight.load(std::sync::atomic::Ordering::Relaxed);
-					info!("Transactions in flight: {:?}", in_flight);
-					if in_flight > 2^16 {
-						info!("Shedding load.");
+					let in_flight = self.transactions_in_flight.load(std::sync::atomic::Ordering::SeqCst);
+					info!(
+						target: "movement_timing",
+						in_flight = %in_flight,
+						"transactions_in_flight"
+					);
+					if in_flight > 2u64.pow(16) {
```

Comment:
Does the compiler optimize `2u64.pow(16) `to its value. If not we can define a const, so the calculus is done one time.

Reply:
Yes, `pow` is a const fn so it's computed at compile time. Being an old programmer I'd use `1 << 16` though 😁